tools/samba/illumina_suite/figaro.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="figaro" name="Figaro" version="@TOOL_VERSION_FIGARO@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>Identify optimal optimizing microbiome rRNA gene trimming parameters for DADA2</description>
+    <xrefs>
+        <xref type="bio.tools"></xref>
+    </xrefs>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="figaro_edam"/>
+    <expand macro="figaro_requirements"/>
+    <required_files>
+        <include type="literal" path="macro.xml"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[
+        RAW_DATA_DIR='$raw_data'
+        AMPLICON_LENGTH='$amplicon_length'
+        F_PRIMER='$f_primer'
+        R_PRIMER='$r_primer'
+        FIGARO_OUTPUT='./output_dir'
+        LOGCMD='./log_dir'
+
+        # Run FIGARO
+        CMD="figaro -i ${RAW_DATA_DIR} -o ${FIGARO_OUTPUT} -a ${AMPLICON_LENGTH} -f ${LENGTH_PRIMER_F} -r ${LENGTH_PRIMER_R}"
+        echo ${CMD} > ${LOGCMD}
+        eval ${CMD}
+
+    ]]></command>
+    <inputs>
+        <param name="raw_data" type="data" format="" label="Raw data" help="TODO" />
+        <param name="amplicon_length" type="integer" value="0"  label="Amplicon length" help="TODO" />
+        <param name="f_primer" type="integer" value="0" label="F Primer" help="TODO" />
+        <param name="r_primer" type="integer" value="0"  label="R Primer" help="TODO" />
+    </inputs>
+    <outputs>
+        <data name="Figaro output" format="json" from_work_dir="./output_dir/*.json" label="${tool.name} on ${on_string} output" />
+        <data name="Figaro logs" format="" from_work_dir="./log_dir/*" label="${tool.name} on ${on_string} log" />
```

Comment:
Do we really want the logs ? (I think for the final version this should not be there)